### ISSUE TITLE
Obsolete information about show_exceptions

### ISSUE THREAD

**Issue by travisp** [getsentry/raven-ruby#371]
```
You'll still want to ensure you've disabled anything that would prevent errors from being propagated to the Raven::Rack middleware, like ActionDispatch::ShowExceptions:

config.action_dispatch.show_exceptions = false # this is the default setting in production
```
https://github.com/getsentry/raven-ruby/wiki/Integrations

show_exceptions is default true in production in rails (at least the latest versions). Is there anything that should be done in modern rails to make sure errors are propagated to Raven?

**Comment by kkumler** [comment]
@travisp I was curious about this too.
It looks like, with the default Raven.configuration.catch_debugged_exceptions = true, it will still capture those shown or debugged by ActionDispatch: https://github.com/getsentry/raven-ruby/blob/9b3ad788130f502ee66b5c4a6a4521fc7e15ecf9/lib/raven/integrations/rails.rb

**Comment by nateberkopec** [comment]
Hey - the answer is no, you shouldn't do anything extra. There's a problem with how we interact with ShowExceptions right now which will be fixed by #422.
